.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
```

Comment:
You're right, so I removed only PHP 5.3 and PHP 5.4. We should really decide what to do, we can't wait forever for a decision on the topic